src/classes/ADDR_CopyAddrHHObjBTN_CTRL.cls

```diff
@@ -318,6 +330,9 @@ public with sharing class ADDR_CopyAddrHHObjBTN_CTRL {
             if (!perms.canUpdate(Contact.SObjectType, HH_BY_CONTACT_ADDR_FIELDS.keySet())) {
                 return false;
             }
+            if  (!perms.canCreate(Address__c.sObjectType, new Set<SObjectField>(ADDRESS_BY_CONTACT_ADDR_FIELDS.values()))) {
```

Comment:
- The address can be updated or created when it is being copied from a Contact or Household, so should this check go outside of the `if/else` condition?
- The `ADDRESS_BY_CONTACT_ADDR_FIELDS` can be a `List` since the `keySet()` is not used anywhere.
- If the user has Create but no Edit permission on the Address, should the error be displayed?